Makefile

```diff
@@ -81,7 +90,13 @@ lint:
 test:
 	go test -race ./...
 
-img/%.png: %.dot
+update-deps:
+	glide update --strip-vcs --strip-vendor --update-vendored --delete
+	glide-vc --only-code --no-tests
+	# see http://sed.sourceforge.net/sed1line.txt
+	for f in $$(find vendor -type f); do sed -i -e :a -e '/^\n*$$/{$$d;N;ba' -e '}' $$f; done
```

Comment:
Probably better to use find's `-exec` instead of the for loop (`-execdir` is safer, but [not in POSIX][1]).  Although you may have to keep the for loop, because `-i` is [not in POSIX][2] for sed.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/find.html
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html